skater/camera.py

```diff
@@ -1,64 +1,61 @@
 
 class Camera:
-
     def __init__(self):
         self.x = 0
-        self.y = 0
-        
+        self.y = 0        
 
     def adjust(self, screen, player):
         
-        LAG = 0.99
+        LAG = 0.01
 
         self.adjust_x(screen, player, LAG)
         self.adjust_y(screen, player, LAG)
 
 
     def adjust_x(self, screen, player, lag):
         """
-        Changes camera x parameteres depending on the location of the player on screen x axis
+        Computes the required camera adjustment depending on player's location and camera's current x shift
         """
-        MARGIN_LEFT = 0.2
-        MARGIN_RIGHT = 0.5
+        margin_left = 0.2
+        margin_right = 0.3
```

Comment:
a czemuz malymi, chyba jak sa constant to sie kapitalami pisze?

Reply:
tak w ogole trzeba zmieinc nazwy tych marginow, bo teraz to nie jest margin od krawedzi tylko od srodka ekranu

Reply:
Więc kapitałami pisze się tylko stałe statyczne (wykraczające poza jedną funkcję), tj. na poziomie klasy / modułu.

Gdybyśmy mieli stałą przypisaną do klasy (jak np `LAG`) -> piszemy wielkimi literami.
Jeśli coś jest tylko wartością lokalną dla funkcji (tj. zdefiniowaną wewnątrz funkcji) -> piszemy małymi  / snake_case